src/PowerShellEditorServices.Hosting/Commands/StartEditorServicesCommand.cs

```diff
@@ -349,6 +349,8 @@ private EditorServicesConfig CreateConfigObject()
             var profile = (PSObject)GetVariableValue("profile");
 
             var hostInfo = new HostInfo(HostName, HostProfileId, HostVersion);
+            var iss = Runspace.DefaultRunspace.InitialSessionState;
+            iss.LanguageMode = Runspace.DefaultRunspace.SessionStateProxy.LanguageMode;
```

Comment:
This fixed the broken tests @rjmholt @andschwa 
Presumably the tests were changing the languagemode after the runspace was created so I set the LanguageMode to the LanguageMode on the SessionStateProxy (which is where it was coming from originally) and normal functionality was restored.